### DISCUSSION TITLE
Got locked and no way to check dashboard

### DISCUSSION THREAD

**Discussion by linegel** [plausible/analytics#1645] [Anything Else]
![image](https://user-images.githubusercontent.com/1746296/151449968-510a6b44-aba7-48c7-aa7b-a33bf93e65e3.png)

**Comment by metmarkosaric** [comment]
that's weird! sorry about that @linegel! could you please email our customer support hello@plausible.io as that's the best line for these types of questions? thanks!

**Comment by metmarkosaric** [comment]
Quick update to confirm that your dashboard has been unlocked now. Thanks again!
